application.py

```diff
@@ -17,112 +18,175 @@
 # AWS Elastic Beanstalk looks for application by default,
 # if this variable (application) isn't set you will get a WSGI error.
 application = app.server
-
-# Customize this layout to include Google Analytics
-gtag_id = os.getenv("GTAG_ID", default="")
-app.index_string = f"""
-<!DOCTYPE html>
-<html>
-    <head>
-        <!-- Global site tag (gtag.js) - Google Analytics -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id={gtag_id}"></script>
-        <script>
-          window.dataLayer = window.dataLayer || [];
-          function gtag(){{dataLayer.push(arguments);}}
-          gtag('js', new Date());
-
-          gtag('config', '{gtag_id}');
-        </script>
-        {{%metas%}}
-        <title>{{%title%}}</title>
-        <meta charset="utf-8"/>
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <!-- Schema.org markup for Google+ -->
-        <meta itemprop="name" content="{luts.title}">
-        <meta itemprop="description" content="{luts.description}">
-        <meta itemprop="image" content="{luts.preview}">
-
-        <!-- Twitter Card data -->
-        <meta name="twitter:card" content="summary_large_image">
-        <meta name="twitter:site" content="@SNAPandACCAP">
-        <meta name="twitter:title" content="{luts.title}">
-        <meta name="twitter:description" content="{luts.description}">
-        <meta name="twitter:creator" content="@SNAPandACCAP">
-        <!-- Twitter summary card with large image must be at least 280x150px -->
-        <meta name="twitter:image:src" content="{luts.preview}">
-
-        <!-- Open Graph data -->
-        <meta property="og:title" content="{luts.title}" />
-        <meta property="og:type" content="website" />
-        <meta property="og:url" content="{luts.url}" />
-        <meta property="og:image" content="{luts.preview}" />
-        <meta property="og:description" content="{luts.description}" />
-        <meta property="og:site_name" content="{luts.title}" />
-
-        <link rel="alternate" hreflang="en" href="{luts.url}" />
-        <link rel="canonical" href="{luts.url}"/>
-        {{%favicon%}}
-        {{%css%}}
-    </head>
-    <body>
-        {{%app_entry%}}
-        <footer>
-            {{%config%}}
-            {{%scripts%}}
-            {{%renderer%}}
-        </footer>
-    </body>
-</html>
-"""
-
+app.index_string = luts.index_string
 app.title = luts.title
 app.layout = layout
 
 
+def get_title_components(years, community, precip_type):
+    year_min = str(years.min())
+    year_max = str(years.max())
+    community_name = str(
+        luts.communities.loc[luts.communities["stid"] == int(community)]["name"].values[
```

Comment:
sometimes Pandas makes it gross to get a single value from a cell, best I can tell